zeppelin-plugins/pom.xml

```diff
@@ -25,10 +25,8 @@
         <artifactId>zeppelin</artifactId>
         <groupId>org.apache.zeppelin</groupId>
         <version>0.10.0-SNAPSHOT</version>
-        <relativePath>..</relativePath>
     </parent>
 
-    <groupId>org.apache.zeppelin</groupId>
```

Comment:
When you clean up the pom files, you should also delete the versions.

>Alternatively, if you want the groupId or the version of your modules to be the same as their parents, you can remove the groupId or the version identity of your module in its POM.
https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Inheritance

Reply:
Upon checking, only the version value of "r/pom.xml" is "0.9.0-SNAPSHOT". Can I understand that it is missing from the version update changes?

Reply:
The r module is deprecated.
https://github.com/apache/zeppelin/commit/b62c6078d1d6dee30f9e35e10bf7e6d0f1e1ffc5

Reply:
Maybe it's time to remove the module.
https://github.com/Reamer/zeppelin/commit/1298e2619d168647806c09a913a69a5bc8883c8b
@zjffdu what are your thinking?

Reply:
I have deleted duplicated version tags of sub modules

Reply:
PR #4145 deletes the R module.

Reply:
@cuspymd I removed the deprecated r module, please rebase your PR.